tests/admin_views/tests.py

```diff
@@ -4899,6 +4900,25 @@ def test_change_form_renders_correct_null_choice_value(self):
         response = self.client.get(reverse('admin:admin_views_choice_change', args=(choice.pk,)))
         self.assertContains(response, '<div class="readonly">No opinion</div>', html=True)
 
+    def test_readonly_foreignkey(self):
+        """
+        Test that ForeignKey renders as links for readonly fields if the target model
```

Comment:
You should omit prefixes like "Test that" since all tests ensure things. This guideline is from [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).

Reply:
Fixed in my last commit